cli/cmd/completion.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+var completionCmd = &cobra.Command{
+	Use:       "completion [bash|zsh]",
+	Short:     "Shell completion",
+	Long:      "Output completion code for the specified shell (bash or zsh).",
```

Comment:
What about adding a  `source <(conduit completion bash)` copy-and-pastable line?

Reply:
will do. was reluctant to make this too verbose, but then realized:
```
$ kubectl completion -h|wc -l
      37
```